crypto/merkle/tree_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/stretchr/testify/require"
 
 	cmtrand "github.com/cometbft/cometbft/libs/rand"
-	. "github.com/cometbft/cometbft/libs/test"
+	. "github.com/cometbft/cometbft/libs/test" //nolint:revive
```

Comment:
What's this? Couldn't find much online. It seems like it's not a rule of our linter, but a different linter, called `revive`. If this is the case, why don't we use only one linter?

Reply:
Specifically it complained about `dot-imports: should not use dot imports (revive)`.